core/src/main/java/org/lflang/generator/c/CCmakeGenerator.java

```diff
@@ -230,6 +257,27 @@ CodeBuilder generateCMakeCode(
     cMakeCode.pr("target_include_directories(${LF_MAIN_TARGET} PUBLIC include/core/modal_models)");
     cMakeCode.pr("target_include_directories(${LF_MAIN_TARGET} PUBLIC include/core/utils)");
 
+    // post target definition board configurations
+    if (targetConfig.platformOptions.board != null) {
+      switch (targetConfig.platformOptions.platform) {
+        case RP2040:
+          String[] bProps = targetConfig.platformOptions.board.split(":");
```

Comment:
This looks like ad hoc parsing. Will it silently produce wrong behavior if there is a typo, or if there is whitespace where whitespace is not expected?

Reply:
Did some whitespace removal and placed reasonable defaults in case of a formatting mismatch.